### ISSUE TITLE
Re-render SVG when zooming

### ISSUE THREAD

**Issue by allmeta** [nsxiv/nsxiv#285]
See image. nsxiv on the left, chromium on the right.

**Comment by N-R-K** [comment]
Yeah, this has been a known limitation. (n)sxiv uses imlib2 for most image related operation. And imlib2 was never designed to work with vector images, so it uses a raster at loading and then performs all scaling operation on that raster; leading to the blurryness in the picture.

I don't think imlib2 is going to get rewritten/redesigned to properly support vector images anytime soon.
